src/MSALWrapper/AuthFlow/Broker.cs

```diff
@@ -29,7 +30,7 @@ public class Broker : IAuthFlow
         /// <summary>
         /// The silent auth timeout.
         /// </summary>
-        private TimeSpan silentAuthTimeout = TimeSpan.FromMinutes(5);
+        private TimeSpan silentAuthTimeout = TimeSpan.FromSeconds(20);

```

Comment:
Should we also extend our initial `WarningDelay` in `AuthFlowExecutor` to `20 seconds` or more so that we provide enough time for silent auth to timeout before we warn for auth prompt

Reply:
That seems reasonable, but we could probably do it in a subsequent PR.